src/pages/home/HomeBlogPreview.jsx

```diff
@@ -82,7 +83,7 @@ function ReadMore() {
   );
 }
 
-function DesktopBlogPreview({ featuredPosts, allPosts }) {
+function DesktopBlogPreview({ FirstPost, allPosts }) {
   const rightColumnPosts = allPosts?.slice(0, 4);
```

Comment:
```suggestion
  const rightColumnPosts = allPosts?.slice(1, 4);
```
This would also have to change slightly, here and elsewhere